tests/testthat/helper-for-tests.R

```diff
@@ -1,4 +1,4 @@
-# Helper for tests 
+# Helper for tests
 getTestDataFilePath <- function(fileName) {
   dataPath <- file.path(getwd(), "..", "data", fsep = .Platform$file.sep)
```

Comment:
Please use `testthat::test_path("data", fileName)` instead it will target `tests/testthat/data/[filename]`.

As general rule, never use `getwd()` or `setwd()` in a package (and avoid it in general).

Reply:
I will delete the whole file. It is not needed in ospsuite.plots.
I copied it in an early version of my package from the ospsuite.ReportingEngine package.